Merge Staging:  Latest code from dev into stage for deployment

supriyartambewaiin opened the pull request:
## Screenshots
: - No Screenshots

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Introduced design previews integrated with Figma for several components.
  - Added new icon options to enhance visual consistency.

- **Style**
  - Improved spacing, padding, and layout across user interface elements such as dashboards, cards, chats, and sidebars.
  - Enhanced dark theme styling for charts, notifications, progress bars, and toggle controls.

- **Refactor**
  - Streamlined component logic and updated demonstration configurations for improved maintainability.

- **Chores**
  - Updated build and TypeScript configurations.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

saniyasonkaria reviewed (approved):
Changes reviewed and approved.

Dipak7028 committed:
Resolved Bug :1790 Design System Component Features > Alignment and mobile view issues

Resolved Bug :1790 Design System Component Features > Alignment and mobile view issues

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1790-components-Features-alignment-mobile-view-issues

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1790-components-Features-alignment-mobile-view-issues

nitinsonawane123 committed:
commit changes for bug 1731

nitinsonawane123 committed:
Commit changes for checkbox

Dipak7028 committed:
Update rds-comp-data-table.tsx

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1778-components-edition-list-list-is-not-displayed-in-small-mobile-view

AkshayPotphode committed:
Commit changes toast icon and dark theme issue

nitinsonawane123 committed:
Merge pull request #1834 from Wai-Technologies/users/nitin/bug-element-checkbox-issue-fixed-across-the-application

Resolved Bug : Element Checkbox is not clickable for application.

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-1731-element-ratings-colors-of-unrated-items-is-different-than-wf-on-slider

AkshayPotphode committed:
Merge branch 'development' into users/akshay/bug-1483-circle-icon-should-be-as-per-the-figma

nitinsonawane123 committed:
Merge pull request #1833 from Wai-Technologies/users/nitin/bug-1731-element-ratings-colors-of-unrated-items-is-different-than-wf-on-slider

Resolved bug 1731 : Design System > Elements > Ratings > Colors of Unrated items is different than WF on Slider.

AkshayPotphode committed:
commit change

supriyartambewaiin committed:
Merge branch 'development' into users/akshay/bug-1483-circle-icon-should-be-as-per-the-figma

ashoksinarewai committed:
commit code for accordian and button grid css issue

ashoksinarewai committed:
Merge branch 'development' into users/ashok/bug-1740-accordian-effect-not-applied

nknikhilkasar committed:
Update story parameters to exclude unnecessary controls in stepper stories

AkshayPotphode committed:
Merge pull request #1837 from Wai-Technologies/users/akshay/bug-1483-circle-icon-should-be-as-per-the-figma

Resolved Bug 1483 : Elements > Toast > Circle Icon & Dark Theme Should As Per The Figma Design

supriyartambewaiin committed:
Merge branch 'development' into users/ashok/bug-1740-accordian-effect-not-applied

ashoksinarewai committed:
Merge pull request #1838 from Wai-Technologies/users/ashok/bug-1740-accordian-effect-not-applied

Resolved Bug - 1740 Accoridon Hover Effect And Button Grid CSS Issue

supriyartambewaiin committed:
Merge branch 'development' into users/nikhil/bug-1549-missing-checkboxes-in-stepper

nknikhilkasar committed:
Refactor story files to use new StoryObj and Meta types; update Message interface visibility

AkshayPotphode committed:
Commit changes for multilevel menu dark theme issue

AkshayPotphode committed:
Merge branch 'development' into users/akshay/bug-1492-multilevel-menu-dark-theme-issue

AkshayPotphode committed:
commit change

ashoksinarewai committed:
commit code for progress bar  css issue

Dipak7028 committed:
Update rds-comp-reviews.tsx

Dipak7028 committed:
Update rds-comp-reviews.tsx

nitinsonawane123 committed:
Commit Changes for bug 1780

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1780-components-feeds&file-mover-thumb-issue-and-folder-icon-missing

Dipak7028 committed:
Update rds-comp-address-input.tsx

supriyartambewaiin committed:
Completed Task Configuration Change in tsconfig file

Pundit-Assistant committed:
Updated Export to Export Type on index.ts

Pundit-Assistant committed:
Merge pull request #1846 from Wai-Technologies/users/supriya/configuration-in-tsconfig-file

Completed Task : Configuration Change in tsconfig file for Single Page Application Deployment

rameshwarkamandar committed:
As per suggestion made changes sidebar

Dipak7028 committed:
Resolved issues for  Responsiveness

Sachin-Ghorpade committed:
Committed changes on code connect

supriyartambewaiin committed:
Merge branch 'development' into users/nikhil/bug-1549-missing-checkboxes-in-stepper

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-1780-components-feeds&file-mover-thumb-issue-and-folder-icon-missing

supriyartambewaiin committed:
Merge branch 'development' into users/ashok/bug-1727-unable-to-see-percentage-on-progress-bar-and-color-visibility

supriyartambewaiin committed:
Merge branch 'development' into users/akshay/bug-1492-multilevel-menu-dark-theme-issue

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1784-components-address-input-Alignment-ssue-in-small-and-large-mobile-views

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1473-components-Review-Enhancement-as-per-design-Kit

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1778-components-edition-list-list-is-not-displayed-in-small-mobile-view

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1790-components-Features-alignment-mobile-view-issues

rameshwarkamandar committed:
Merge pull request #1847 from Wai-Technologies/users/rameshwar/1701-Create-new-component-Side-Bar-as-per-design-Chat

Bug 1701: Resolved Issue on Design System > Component > Create new component Side Bar as per design [Chat]

AkshayPotphode committed:
Merge branch 'development' into users/akshay/bug-1492-multilevel-menu-dark-theme-issue

AkshayPotphode committed:
Merge branch 'users/akshay/bug-1492-multilevel-menu-dark-theme-issue' of https://github.com/Wai-Technologies/raaghu-react into users/akshay/bug-1492-multilevel-menu-dark-theme-issue

AkshayPotphode committed:
Merge branch 'development' into users/akshay/bug-1492-multilevel-menu-dark-theme-issue

AkshayPotphode committed:
Merge pull request #1840 from Wai-Technologies/users/akshay/bug-1492-multilevel-menu-dark-theme-issue

Resolved Bug 1492 : Multilevel Menu Dark Theme Issue

Sachin-Ghorpade committed:
Merge branch 'development' into users/sachin/bug-1619-design-system-elements-code-connect-changes-tooltip-top-nav

Sachin-Ghorpade committed:
Merge pull request #1852 from Wai-Technologies/users/sachin/bug-1619-design-system-elements-code-connect-changes-tooltip-top-nav

Completed Task 1619: Design System > Elements > Code connects changes for Video Player - Truncated Text - Tree Structure - Tooltip - Toast elements

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1778-components-edition-list-list-is-not-displayed-in-small-mobile-view

Dipak7028 committed:
Merge pull request #1835 from Wai-Technologies/users/dipak/bug-1778-components-edition-list-list-is-not-displayed-in-small-mobile-view

Resolved Bug :1778 Design System Component Edition List is displayed in small mobile view

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1473-components-Review-Enhancement-as-per-design-Kit

Dipak7028 committed:
Merge pull request #1842 from Wai-Technologies/users/dipak/bug-1473-components-Review-Enhancement-as-per-design-Kit

Resolved bug 1473: Design System Component Review Enhancement as per design Kit

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1784-components-address-input-Alignment-ssue-in-small-and-large-mobile-views

Dipak7028 committed:
Merge pull request #1844 from Wai-Technologies/users/dipak/bug-1784-components-address-input-Alignment-ssue-in-small-and-large-mobile-views

Resolved bug 1784: Design System Components Address Input Alignment issue in small and large mobile views

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1642-components-admin-dashboard

Dipak7028 committed:
Merge pull request #1850 from Wai-Technologies/users/dipak/bug-1642-components-admin-dashboard

 Resolved Bug 1642: Design System Component Admin Dashboard Responsiveness Issue      Resolved issues for  Responsiveness

supriyartambewaiin committed:
Merge branch 'development' into users/nikhil/bug-1549-missing-checkboxes-in-stepper

Sachin-Ghorpade committed:
Committed changes on code connect for Reviews element

nknikhilkasar committed:
Merge pull request #1839 from Wai-Technologies/users/nikhil/bug-1549-missing-checkboxes-in-stepper

Resolved Bug 1549: Design System > Elements > Stepper > Not able to see dash in dark theme and missing check boxes

rameshwarkamandar committed:
Export icon arrow should display proper and close icon should align properly

ashoksinarewai committed:
commit code for grid line dark theme issue.

ashoksinarewai committed:
Merge pull request #1855 from Wai-Technologies/users/ashok/bug-grid-line-not-visible-in-dark-theme

Resolved Bug 1720 - Grid Line Not Visible In Dark Theme

rameshwarkamandar committed:
Icons is not showing properly

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1790-components-Features-alignment-mobile-view-issues

Dipak7028 committed:
Update rds-comp-features.tsx

AkshayPotphode committed:
Commit for dropdown open proper on second click

Dipak7028 committed:
Merge pull request #1823 from Wai-Technologies/users/dipak/bug-1790-components-Features-alignment-mobile-view-issues

Resolved Bug :1790 Design System Component Features > Alignment and mobile view issues

rameshwarkamandar committed:
use specific class name

supriyartambewaiin committed:
Merge branch 'development' into users/akshay/bug-1491-dropdown-is-not-open-proper-on-second-click

nknikhilkasar committed:
Merge branch 'development' into users/rameshwar/1513-Element-Enhancement-in-file-uploader

rameshwarkamandar committed:
Merge pull request #1854 from Wai-Technologies/users/rameshwar/1513-Element-Enhancement-in-file-uploader

Bug 1513: Resolved Issue on Element > File uploader >Export icon arrow should display proper and close icon should align properly.

rameshwarkamandar committed:
Action column three dots button is not display

nknikhilkasar committed:
Merge branch 'development' into users/akshay/bug-1491-dropdown-is-not-open-proper-on-second-click

AkshayPotphode committed:
Commit changes for chat issues

Sachin-Ghorpade committed:
Committed changes on i letter in divider is not according to Figma

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1780-components-feeds&file-mover-thumb-issue-and-folder-icon-missing

nitinsonawane123 committed:
Merge pull request #1843 from Wai-Technologies/users/nitin/bug-1780-components-feeds&file-mover-thumb-issue-and-folder-icon-missing

Resolved Bug 1780 & 1722 :  Design System > Components > Feeds/File Mover > Thumb alignment issue and folder icon is missing &  Design System > Components > Like and dislike > Alignment issue of like and dislike button

supriyartambewaiin committed:
Merge branch 'development' into users/akshay/bug-1807-chat-should-be-as-per-the-figma

supriyartambewaiin committed:
Merge branch 'development' into users/akshay/bug-1491-dropdown-is-not-open-proper-on-second-click

AkshayPotphode committed:
Merge pull request #1859 from Wai-Technologies/users/akshay/bug-1807-chat-should-be-as-per-the-figma

Resolved Bug 1807 : Resolve Chat Issues

AkshayPotphode committed:
Merge branch 'development' into users/akshay/bug-1491-dropdown-is-not-open-proper-on-second-click

supriyartambewaiin committed:
Merge branch 'development' into users/rameshwar/1511-Components-Data-Table-Enhancement-as-per-Figma-Design

supriyartambewaiin committed:
Merge branch 'development' into users/sachin/bug-1707-design-system-elements-divider-i-letter-in-divider-is-not-according-to-figma

rameshwarkamandar committed:
Merge pull request #1858 from Wai-Technologies/users/rameshwar/1511-Components-Data-Table-Enhancement-as-per-Figma-Design

Bug 1511: Resolved Issue on Design System > Components > Data Table > Enhancement as per Figma Design.

AkshayPotphode committed:
Merge branch 'development' into users/akshay/bug-1491-dropdown-is-not-open-proper-on-second-click

AkshayPotphode committed:
Merge pull request #1856 from Wai-Technologies/users/akshay/bug-1491-dropdown-is-not-open-proper-on-second-click

Resolved Bug 1491 : Dropdown Should Open Proper Click on Second Time

supriyartambewaiin committed:
Merge branch 'development' into users/sachin/bug-1707-design-system-elements-divider-i-letter-in-divider-is-not-according-to-figma

Sachin-Ghorpade committed:
Merge pull request #1860 from Wai-Technologies/users/sachin/bug-1707-design-system-elements-divider-i-letter-in-divider-is-not-according-to-figma

Resolved Bug 1707 : Design System > Elements >Divider >i letter in divider is not according to Figma

supriyartambewaiin committed:
Merge branch 'development' into users/sachin/bug-1619-design-system-elements-code-connect-changes-for-reviews

Sachin-Ghorpade committed:
Merge pull request #1857 from Wai-Technologies/users/sachin/bug-1619-design-system-elements-code-connect-changes-for-reviews

Completed Task 1619: Design System > Elements > Code connects changes for Reviews element

Sachin-Ghorpade committed:
committed changes on off canvas need fix issue

supriyartambewaiin committed:
Merge branch 'development' into users/sachin/bug-1471-design-system-elements-off-canvas-cross-button-color-issue-and-text-is-different-than-figma

Sachin-Ghorpade committed:
Merge pull request #1861 from Wai-Technologies/users/sachin/bug-1471-design-system-elements-off-canvas-cross-button-color-issue-and-text-is-different-than-figma

Resolved Bug 1471 : Off Canvas > Not fixed on Prod > Cross button color issue and text is different from figma

ashoksinarewai committed:
commit code for ai prompt box css realted

ashoksinarewai committed:
Merge branch 'development' into users/ashok/bug-ai-prompt-box-alignment-issue

ashoksinarewai committed:
Merge branch 'development' into users/ashok/bug-1727-unable-to-see-percentage-on-progress-bar-and-color-visibility

nitinsonawane123 committed:
Commit Changes for bug 708

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-708-element-counter-ui-issue

Dipak7028 committed:
Resolved Bug 1741: Design System Elements Card Text Not Displayed in Mobile Views

nitinsonawane123 committed:
Commit Changes for build error

nitinsonawane123 committed:
Merge pull request #1865 from Wai-Technologies/users/nitin/bug-708-element-counter-ui-issue

Resolved bug 708 : Design System > Element > Counter > Enhancement as per the Figma Design.

supriyartambewaiin committed:
Merge branch 'development' into users/ashok/bug-ai-prompt-box-alignment-issue

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1741-elements-card-text-not-displayed-in-mobile-view

ashoksinarewai committed:
Merge pull request #1862 from Wai-Technologies/users/ashok/bug-ai-prompt-box-alignment-issue

Resolved Bug - AI Prompt Box Responsive issue

ashoksinarewai committed:
commit code in progress bar css

ashoksinarewai committed:
Merge branch 'development' into users/ashok/bug-1727-unable-to-see-percentage-on-progress-bar-and-color-visibility

Dipak7028 committed:
Update rds-big-number.tsx

ashoksinarewai committed:
Merge pull request #1841 from Wai-Technologies/users/ashok/bug-1727-unable-to-see-percentage-on-progress-bar-and-color-visibility

Resolved Bug 1727 - Progress Bar Percentage and Visibilty Issue

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1741-elements-card-text-not-displayed-in-mobile-view

Dipak7028 committed:
Merge pull request #1864 from Wai-Technologies/users/dipak/bug-1741-elements-card-text-not-displayed-in-mobile-view

Resolved Bug 1741: Design System Elements Card Text Not Displayed in Mobile Views

Sachin-Ghorpade committed:
Committed changes on radio button issue

Sachin-Ghorpade committed:
Merge pull request #1868 from Wai-Technologies/users/sachin/bug-1495-design-system-elements-radio-ui-is-not-same-as-per-figma

Resolved Bug 1495: Design System > Element > Radio > Radio button UI Issues

supriyartambewaiin committed:
Merge branch 'staging' into users/supriya/merge-latest-dev-to-stage-deployment-task

nitinsonawane123 reviewed (approved):
Approved
